.travis.yml

```diff
@@ -23,6 +23,16 @@ matrix:
     env: TOXENV=py36-django18 DEPLOY=1
   - python: "3.7"
     env: TOXENV=py37-django18
+  - python: "2.7"
+    env: TOXENV=py27-django111
+  - python: "3.4"
+    env: TOXENV=py34-django111
+  - python: "3.5"
+    env: TOXENV=py35-django111
```

Comment:
I don't mind too much, but I think it's not necessary to fill the entire matrix here (i.e. testing all of Python 2.7, 3.4, 3.5, 3.6, 3.7 on both Django versions).  How slow is the CI now?  It's acceptable to me either way, but you should probably consider leaving some gaps (like testing python 3.4 only on Django 1.8, 3.5 only on 1.11, 3.6 only on 1.8 etc.)

The chance of a bug affecting for instance *only* python 3.5 and Django 1.11, and not any other combination of Python and Django versions, seems very low.  So feel free to make this a bit more sparse if you feel CI could be faster.

Reply:
> How slow is the CI now?

It's apparently about 16 minutes ( https://travis-ci.org/github/release-engineering/kobo/builds/672842771 ).

It's subjective of course, but to me this is a bit on the slow side, so I would be in favor of making this more sparse as suggested.

Reply:
I have reduced it.